contracts/axelar-soroban-interfaces/Cargo.toml

```diff
@@ -8,4 +8,5 @@ crate-type = ["rlib"]
 
 [dependencies]
 axelar-soroban-std = { workspace = true }
+hex-literal = "0.3"
```

Comment:
this is a dev dependency, since it's only used for tests

Reply:
Good call, moved under dev dependency